e2e_samples/parking_sensors/infrastructure/modules/datafactory.bicep

```diff
@@ -0,0 +1,51 @@
+param project string
+@allowed([
+  'dev'
+  'stg'
+  'prod'
+])
+param env string
+param location string = resourceGroup().location
+param deployment_id string
+
+// param account_name string = ''
+// param repository_name string = ''
+// param collaboration_branch string = 'main'
+// param root_folder string = '/e2e_samples/parking_sensors/adf'
+
+resource datafactory 'Microsoft.DataFactory/factories@2018-06-01' = {
+  name: '${project}-adf-${env}-${deployment_id}'
+  location: location
+  tags: {
+    DisplayName: 'Data Factory'
+    Environment: env
+  }
+  identity: {
+    type: 'SystemAssigned'
+  }
+}
+
```

Comment:
I know the code is commented, just make a comment saying that this piece will enable the automation of the Git integration for the Dev environment "when" and "if" the Import checkbox will become available.

Reply:
I actually removed the commented code just so its not cluttering the script. We can always go back to git history to retrieve it once the import checkbox can be automated :)